docs/dev/README.md

```diff
@@ -55,6 +55,42 @@ To run e2e tests for the repository:
 make e2e-cli-core
 ```
 
+### Environment variables for testing the CLI
+
+Some test options affecting the CLI are only available through the use of environment
+variables.  Below is the list of such test variables:
```

Comment:
Should we add additional line stating that these variables are only for testing and can be updated or removed without any prior notice (or something like that)?

Reply:
How about "These variables are only meant to be used during testing and are not subject to the CLI's backwards-compatibility policy, i.e., the could be changed or removed at any time."?